header/p2p/exchange_test.go

```diff
@@ -230,11 +241,18 @@ func TestExchange_RequestByHashFails(t *testing.T) {
 	require.Equal(t, resp.StatusCode, p2p_pb.StatusCode_NOT_FOUND)
 }
 
-func createMocknet(t *testing.T) (libhost.Host, libhost.Host) {
-	net, err := mocknet.FullMeshConnected(2)
+func Test_PrepareRequests(t *testing.T) {
```

Comment:
can you move this into a file called `session_test` since it tests code in session.go